components/motor/fake/motor.go

```diff
@@ -30,8 +41,8 @@ type PinConfig struct {
 
```

Comment:
I can't comment on line 38 directly, but I'm pretty sure `Direction` is not used anywhere, and is dead code that can be removed. and if my other comment about it being safe to remove PWM is right, this is all dead code.

I wonder if this was copied from a real motor, and incorrectly left in even though a fake motor shouldn't need any of it.

Reply:
probably was. I wonder if it's used anywhere for testing since fake motor is used sometimes. I can look into all the unused code